subprojects/logging/src/testFixtures/groovy/org/gradle/internal/logging/text/TestStyledTextOutput.groovy

```diff
@@ -16,10 +16,9 @@
 
 package org.gradle.internal.logging.text
 
-import org.gradle.internal.SystemProperties
 import org.gradle.internal.logging.text.StyledTextOutput.Style
 
-class TestStyledTextOutput extends AbstractStyledTextOutput {
+class TestStyledTextOutput extends AbstractLineChoppingStyledTextOutput {
```

Comment:
You got a valid point. I did some more thinking and I created `TestLineChoppingStyledTextOutput` that delegate to `TestStyledTextOuput`.